rdf-star-cg-spec.html

```diff
@@ -829,9 +829,66 @@ <h2>SPARQL* Update</h2>
   <section>
     <h2>RDF* Semantics</h2>
 
-    <p>In this section, we provide a model-theoretic semantics for RDF*, by extending the one defined in [[[RDF11-MT]]] [[RDF11-MT]].</p>
+    <p>In this section, we provide a model-theoretic semantics for RDF*, based on the one defined in [[[RDF11-MT]]] [[RDF11-MT]]. More precisely, we define a mapping from RDF*'s abstract syntax into standard RDF's abstract syntax, and define the semantics of <a>RDF* graphs</a> in terms of the semantics of the mapped <a>RDF graphs</a>.</p>
+
+    <p>In the following, we introduce a number of RDF*-specific definitions, which rely on the following notions, defined in [[[RDF11-CONCEPTS]]] [[RDF11-CONCEPTS]] and [[[RDF11-MT]]] [[RDF11-MT]]:
+      <dfn data-cite="RDF11-CONCEPTS#dfn-datatype">datatype</dfn>,
+      <dfn data-cite="RDF11-CONCEPTS#dfn-lexical-form">lexical form</dfn>,
+      <dfn data-cite="RDF11-MT#dfn-ill-typed">ill-typed</dfn>,
+      <dfn data-cite="RDF11-MT#dfn-merging" data-lt="merge">merging</dfn>,
+      <dfn data-cite="RDF11-MT#dfn-satisfiable" data-lt="satisfiable">satisfiability</dfn>,
+      <dfn data-cite="RDF11-MT#dfn-unsatisfiable" data-lt="unsatisfiable">unsatisfiability</dfn>,
+      <dfn data-cite="RDF11-MT#dfn-entail" data-lt="entail|entailed">entailment</dfn>,
+      and <dfn data-cite="RDF11-MT#dfn-equivalent" data-lt="equivalent">equivalence</dfn>.
+    </p>
+
+    <section>
+      <h2>Mapping RDF* abstract syntax to RDF</h2>
+
+      <p>We consider six <a>IRIs</a> |ST:subject term|, |PT:predicate term|, |OT:object term|, |SS:subject string|, |PS:predicate string| and |OS:object string| that will have a special meaning in our mapping.</p>
+
+      <p>We define a mapping |L| that maps any term |t| that is either an <a>IRI</a> or a <a>literal</a> to a literal with <ul>
+        <li>`xsd:string` as its the <a>datatype</a>, and</li>
+        <li>the <a data-cite="N-TRIPLES#canonical-ntriples">canonical N-Triples</a> representation of |t| as its <a>lexical form</a> [[N-TRIPLES]]. If `t` is itself a literal with the `xsd:string` <a>datatype</a>, the representation MUST NOT include the datatype.</li>
+      </ul>
+      </p>
+
+      <p>Given an <a>RDF* graph</a> |G|, the following steps transform it into an <a>RDF graph</a> that we call |unstar|(|G|).</p>
+
+      <ol>
+        <li>Pick an <a>RDF* triple</a> (|s|, |p|, |o|) in the <a>constituents</a> |G| such that neither |s|, |p| nor |o| is an <a>embedded triple</a>.</li>
```

Comment:
```suggestion
        <li>Pick an <a>RDF* triple</a> (|s|, |p|, |o|) in the <a>constituent</a> |G| such that neither |s|, |p| nor |o| is an <a>embedded triple</a>.</li>
```

Reply:
Actually that should be "in the constituents of G". Thanks for spotting it. It's fixed in my local copy and will be fixed in the next commit.